receiver/prometheusexecreceiver/subprocessmanager/manager_test.go

```diff
@@ -109,6 +109,33 @@ func TestRun(t *testing.T) {
 			wantElapsed: 4 * time.Millisecond,
 			wantErr:     false,
 		},
+		{
+			name: "sleep 4ms and fail",
+			process: &SubprocessConfig{
+				Command: "go run testdata/test_crasher.go 4 1",
+				Env:     []EnvConfig{},
+			},
+			wantElapsed: 4 * time.Millisecond,
+			wantErr:     true,
+		},
+		{
+			name: "sleep the default time (because broken value with spaces) and succeeds",
+			process: &SubprocessConfig{
+				Command: "go run testdata/test_crasher.go \"1 2 3\" 0",
+				Env:     []EnvConfig{},
+			},
+			wantElapsed: 2 * time.Millisecond,
+			wantErr:     false,
```

Comment:
Nit: do we care about this case? This seems to verify that `test_crasher.go` correctly implements invalid command line. I think this is superfluous, we are essentially testing the test code :-)

Reply:
Yes this case is important to determine whether the `"1 2 3"` argument is processed as 1 single argument or not, and therefore that `0` is processed as the 2nd argument (which determines the exit code of **test_crasher.go**.

This test will fail on Windows without the change introduced in this PR.

Reply:
Ah, I see, I misunderstood what the test does. Perhaps add a comment to explain, it was not obvious to me.

Reply:
Added comments in a new commit. All tests seem to pass ok now, thank you!